aviatrix/resource_aviatrix_spoke_gateway.go

```diff
@@ -1787,6 +1838,32 @@ func resourceAviatrixSpokeGatewayUpdate(d *schema.ResourceData, meta interface{}
 		GwName:    d.Get("gw_name").(string),
 	}
 
+	commSendCurr, commAcceptCurr, err := client.GetGatewayBgpCommunities(gateway.GwName)
+	if d.HasChange("bgp_accept_communities") {
+		acceptComm, ok := d.Get("bgp_accept_communities").(bool)
+		if !ok {
+			return fmt.Errorf("failed to assert bgp_accept_communities as a boolean")
+		}
+		if acceptComm != commAcceptCurr || err != nil {
+			err := client.SetGatewayBgpCommunitiesSend(gateway.GwName, acceptComm)
```

Comment:
same question as above for both this call and the error message below and the places where this is done below.